src/App.js

```diff
@@ -186,6 +187,7 @@ function Root() {
   return (
     <Routes>
       <Route path="/nano_contract" element={<StartedComponent children={ <NanoContractList />} loaded={true} />} />
+      <Route path="/nano_contract/select_blueprint/" element={<StartedComponent children={ <NanoContractSelectBlueprint />} loaded={true} />} />
```

Comment:
Why not `nano_contract/create`?

Reply:
As discussed [here](https://github.com/HathorNetwork/hathor-wallet/pull/529#discussion_r1624726562), the reason of the name is because we might want to use this screen elsewhere.